lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
```

Comment:
the doctype lies! since we're shifting away from displaying high precision scores is this still necessary? what about a `precision: binary|grade|numeric` property or something for each audit and we can say goodbye to `typeof`